src/cmd/linuxkit/run_hyperkit.go

```diff
@@ -42,7 +42,15 @@ func runHyperKit(args []string) {
 	mem := flags.Int("mem", 1024, "Amount of memory in MB")
 	var disks Disks
 	flags.Var(&disks, "disk", "Disk config. [file=]path[,size=1G]")
-	data := flags.String("data", "", "Metadata to pass to VM (either a path to a file or a string)")
+	data := flags.String("data", "", "String of metadata to pass to VM; error to specify both -data and -data-file")
+	dataPath := flags.String("data-file", "", "Path to file containing metadata to pass to VM; error to specify both -data and -data-file")
+
+	if *data != "" && *dataPath != "" {
+		fmt.Println("Cannot specify both -data and -data-file")
```

Comment:
minor nit, but I would just use `log.Fatal()` here. We do this on other errors as well.

Reply:
Updated